nvflare/app_common/executors/exec_task_fn_wrapper.py

```diff
@@ -11,69 +11,67 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import builtins
 import logging
+import os
 import sys
 import traceback
-from typing import Dict
 
-from nvflare.fuel.utils.function_utils import find_task_fn, require_arguments
+print_fn = builtins.print
 
 
-class ExecTaskFuncWrapper:
-    def __init__(self, task_fn_path: str, task_main_args: str = None, task_fn_args: Dict = None):
+class TaskScriptRunner:
```

Comment:
we should rename the filenames (this and the test file) to reflect new class name